modules/docs/arrow-docs/docs/docs/datatypes/basic/README.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+title: Basic Types
+permalink: /docs/datatypes/basic/
+---
+
+## Basic Types
+
+Arrow provides [typeclass]({{ '/docs/patterns/glossary/' | relative_url }}) instances for several platform types.
+These instances are available in the module `arrow-instances`.
+
+### Numbers
+
+- [`Show`]({{ '/docs/typeclasses/show/' | relative_url }})
+
+- [`Eq`]({{ '/docs/typeclasses/eq/' | relative_url }})
+
+- [`Order`]({{ '/docs/typeclasses/order/' | relative_url }})
+
+- [`Semigroup`]({{ '/docs/typeclasses/semigroup/' | relative_url }})
+
+- [`Monoid`]({{ '/docs/typeclasses/monoid/' | relative_url }})
+
+### String
```

Comment:
We should probably think of a way to automate this since maintaining instances like this is going to become a pain and can send users the wrong message of certain instances not being available for certain data types.